projecttacoma/fqm-execution
# Logic highlighting documentation

# Summary
This PR adds more detailed descriptions of the measure logic highlighting to our documentation, specifically, the motivation behind unstyled highlighting.

## New behavior
- No new behavior.

## Code changes
- `README.md` - documentation additions
- `logic-highlighting-example-2.png` - screenshot of the relevant logic highlighting behavior.

# Testing guidance
- Check grammar
- Make sure everything makes sense and looks nice!
